### DISCUSSION TITLE
4.0.0-alpha.1

### DISCUSSION THREAD

**Discussion by ocean90** [GlotPress/GlotPress-WP#1422] [Announcements]
## What's Changed
**Breaking Changes**
* GlotPress now requires PHP 7.4. ([#1417])

**Bugfixes**
* Enhance `gp_levenshtein()` for speed improvements. Props @dd32 ([#1408])
* Highlight current navigation menu item. Props @pedro-mendonca ([#1379])

**Features**
* Improve links to glossaries. Props @pedro-mendonca ([#1375])

**Full Changelog**: https://github.com/GlotPress/GlotPress/compare/3.0.0...4.0.0-alpha.1

<hr /><em>This discussion was created from the release <a href='https://github.com/GlotPress/GlotPress/releases/tag/4.0.0-alpha.1'>4.0.0-alpha.1</a>.</em>

**Comment by psmits1567** [comment]
The current version seems to work properly. Although sometimes fetching records from translation memory is still slow. Would it be possible to reduce the amount of suggestions. I often see suggestions that do not fit at all. So all suggestions that have a score below 80% could be removed.

Or add a parameter to not show suggestions from translation memory
